sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,17 +1,15 @@
 # Release History
 
-## 1.15.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.14.1 (2024-10-10)
```

Comment:
I don't think this change warrants shipping a new out-of-band core this month. We should wait till our regular release cadence.
```suggestion
## 1.15.0-beta.1 (Unreleased)
```

Reply:
Regular release cadence is fine, this can't be released as beta though.

Reply:
Really, that's unfortunate. How come?
Is it because eventhubs betas are only present in the main registry?